docs/DataSources.md

```diff
@@ -144,6 +144,15 @@ We document these configuration options in the next section:
             nonexistent file, or an HTTP resource giving a 404) will result in a <code>resolve</code> with an empty
             payload rather than a <code>reject</code> response.</td>
         </tr>
+        <tr>
+            <td><code>censorRequestOptionsLog</code></td>
+            <td><code>Object</code> (map of <code>String</code> to <code>Boolean</code>) (default: <code>{auth: true}</code>)</td>
```

Comment:
Add `"headers.Authorization": true` into the default.

Reply:
Well caught. Ready for another look